cookbooks/fb_profile/recipes/default.rb

```diff
@@ -27,12 +27,33 @@
 
 directory '/etc/profile.d' do
   owner node.root_user
+  # https://github.com/chef/cookstyle/issues/657
+  # rubocop:disable Lint/UnneededCopDisableDirective
+  # rubocop:disable ChefDeprecations/NodeMethodsInsteadofAttributes
   group node.root_group
+  # rubocop:enable ChefDeprecations/NodeMethodsInsteadofAttributes
+  # rubocop:enable Lint/UnneededCopDisableDirective
   mode '0755'
 end
 
 template '/etc/profile.d/fb_profile.sh' do
   owner node.root_user
+  # https://github.com/chef/cookstyle/issues/657
+  # rubocop:disable Lint/UnneededCopDisableDirective
+  # rubocop:disable ChefDeprecations/NodeMethodsInsteadofAttributes
   group node.root_group
+  # rubocop:enable ChefDeprecations/NodeMethodsInsteadofAttributes
+  # rubocop:enable Lint/UnneededCopDisableDirective
   mode '0644'
 end
+
+# Debian doesn't do the redhat make-sure-non-login-shells-get-aliases
+# So this is the bashrc from debian/ubuntu with that extra bit in there
+if node.debian? || node.ubuntu?
```

Comment:
usually we prefer `only_if` for these

Reply:
sort-of. `only_if` is generally preferred because it's runtime safe. In this case (1) the conditional is compile-time safe and (2) that file shouldn't *look* like it's being managed by Chef on RHEL-like machines, because that'll be confusing as hell to people.